### ISSUE TITLE
DSP Node Hierarchy and Parallel Rendering.

### ISSUE THREAD

**Issue by mitchmindtree** [RustAudio/dsp-chain#10]
The DSP non-cyclical Node graph should allow us to consider rendering our audio in parallel (a task per node). This would be a great feat. in dsp if we could pull it off. There is perhaps a lot of potential for gain in performance with parallel audio, though this would require a lot of benchmarking and fiddling, and probably won't occur until after the main framework of rust-dsp is mostly complete.

**Comment by mitchmindtree** [comment]
Closing in favour of #89
